edu.gemini.lch.services/src/main/java/edu/gemini/lch/services/impl/ConfigurationServiceImpl.java

```diff
@@ -232,15 +225,22 @@ private void processChange(Configuration.Value value) {
      */
     private void updateDefaultTimeZone() {
```

Comment:
I'm not totally sure but I think we want this gone. This sets an offset of the timezone read from the db and it is used for the cases the chilean government changes the tz and we can't update the tzdb like for joda
The JVM keeps its tzdb uptodate (though 1.8 will get dropped of those updates soon)
@swalker2m WDYT?

Reply:
Right, one of the goals is to make responding to the daylight saving updates the same across all of our apps.  It is painful to do the update, but we have to do it for everything else anyway.

Reply:
Yes, agree. I stumbled across this early on in my excavation and only made the connection as to what it was used for later, i.e. managing the Chile wonky DST settings. I'll try to pull it out, as well as the time zone offset field which should no longer serve any purpose.

Reply:
Wait...  as far as I understand it, I think what we want is to get rid of the `Configuration.Value.TIME_ZONE_OFFSET` field, which is what we used to mitigate the Chilean DST oddness. This, we should still:

`TimeZone.setDefault(siteService.getSiteTimeZone());`

to set the local time zone appropriately. (This also handles settling the proper `ZoneId`.)

Thus, we should just have something along the lines of:

```
private void setDefaultTimeZone() {
     TimeZone.setDefault(siteService.getSiteTimeZone());
}
```

WDYT?

Reply:
We should also be able to yoink any UI code that deals with the time zone offset allowances, e.g. `ConfigurationServiceImpl.processChange`, and any `Configuration.Value` code that deals with `TIME_ZONE_OFFSET`.